packages/hub/services/search-index.ts

```diff
@@ -222,6 +232,19 @@ class IndexerRun implements IndexerHandle {
     });
     return out!.code!;
   }
+  private async writeCardDataToIndex(card: RawCard, compiledCard: CompiledCard): Promise<CompiledCard> {
```

Comment:
What did the unit tests reveal as to why was this separate method necessary? it seems like writeToIndex would be sufficient. Regardless, you'll also need to update the `raw` key as well, since it also includes the data

Reply:
There was a unit test for updating a data only card. We simply can’t compile a card that is data only.

Reply:
I wonder if that means we need to merge the data only update into the existing raw card?